modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJSPFSetResultsSetTotalMethodCheck.testjspf

```diff
@@ -8,5 +8,9 @@
 <%
 SearchContainer<Entry> searchContainer = new SearchContainer();
 
-searchContainer.setResultsAndTotal(() -> resultsParameter, totalParameter);
+		try {
+	searchContainer.setResultsAndTotal(() -> resultsParameter, totalParameter);
+		} catch (Throwable e) {
+			throw new PortalException(e);
+		}
```

Comment:
Hi @regiCesar21 the indentation seems to be a little of here. Can you adjust, please?
We should keep the original indentation of the file being edited.